homeassistant/components/velbus/climate.py

```diff
@@ -40,7 +42,7 @@ class VelbusClimate(VelbusEntity, ClimateEntity):
     _attr_preset_modes = list(PRESET_MODES)
 
     @property
-    def target_temperature(self) -> int | None:
+    def target_temperature(self) -> Any:
```

Comment:
This shouldn't return Any. Please check the climate integration parent entity for allowed return values.